src/processor/operator/physical_operator.cpp

```diff
@@ -23,8 +23,11 @@ std::string PhysicalOperatorUtils::operatorTypeToString(PhysicalOperatorType ope
     case PhysicalOperatorType::STANDALONE_CALL: {
         return "STANDALONE_CALL";
     }
-    case PhysicalOperatorType::COPY_TO: {
-        return "COPY_TO";
+    case PhysicalOperatorType::COPY_TO_CSV: {
```

Comment:
We can probably use `LOC` to strip the whole PhysicalTypeToString function from coverage report.